### DISCUSSION TITLE
Add Argos, John Lewis, Smyths and 365games for PS5 search

### DISCUSSION THREAD

**Discussion by MvPolish2573** [jef/streetmerchant#977] [Ideas and feature requests]
I have noticed that the following are missing from retailers for the ps5 is there any chance to add them, links are below

Argos-https://www.argos.co.uk/search/playstation-5/opt/sort:price:desc/

John Lewis-https://www.johnlewis.com/search?search-term=playstation+5&suggestion=true#_search_suggestion_referral

John Lewis (direct to disc version)-https://www.johnlewis.com/sony-playstation-5-console-with-dualsense-controller/white/p5115192

Smyths Ps5-https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/playstation-5/playstation-5-consoles/playstation-5-console/p/191259

Smyths Ps5 DE-https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/playstation-5/playstation-5-consoles/playstation-5-digital-edition-console/p/191430

365games-https://www.365games.co.uk/ps5/page-2

**Comment by MvPolish2573** [comment]
Also currys, box and very are not returning any results at all not even out of stock. amazon-uk, game and ebuyer are working fine and showing out of stock

**Comment by J-Hinton** [comment]
Hi @MvPolish2573,

I can't find the exact URLs for the relevant stores. Would you be able to assist? If you can post the links, then I'm happy to add them.

Thanks
James

**Comment by HTahir96** [comment]
Hi @J-Hinton, I believe only Smyths have a direct link at the moment, the rest have been removed.

**Comment by w33l** [comment]
Here are some UK direct links that are still not on the list. Currently only Amazon, Ebuyer and Game seem to be working.

Hope these help!

Argos Console 
https://www.argos.co.uk/product/6795199

Argos DE
https://www.argos.co.uk/product/6795151

John Lewis Console
https://www.johnlewis.com/sony-playstation-5-console-with-dualsense-controller/white/p5115192

Smyths Console
https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/playstation-5/playstation-5-consoles/playstation-5-console/p/191259

Smyths DE
https://www.smythstoys.com/uk/en-gb/video-games-and-tablets/playstation-5/playstation-5-consoles/playstation-5-digital-edition-console/p/191430

ShopTo Console
https://www.shopto.net/en/ps5hw01-playstation-5-console-p191472/

ShopTo DE
https://www.shopto.net/en/ps5hw02-playstation-5-digital-console-p195341/

Box Console
https://www.box.co.uk/CFI-1015A-Sony-Playstation-5-Console_3199689.html

Box DE
https://www.box.co.uk/CFI-1015B-Sony-PlayStation-5-Digital-Edition-Conso_3199692.html

Zatu Console
https://www.board-game.co.uk/product/playstation-5/

Zatu DE
https://www.board-game.co.uk/product/playstation-5-digital-edition/

**Comment by huntinganswers** [comment]
can we use this thread for the states too?

**Comment by MvPolish2573** [comment]
> Hi @MvPolish2573,
> 
> I can't find the exact URLs for the relevant stores. Would you be able to assist? If you can post the links, then I'm happy to add them.
> 
> Thanks
> James

hi appologies for the late reply someone beet me to it above ^^ they posted all the links there. and yh i believe some of the direct links have been removed but the ones above are still active
